package.json

```diff
@@ -58,22 +58,22 @@
     "i18next-browser-languagedetector": "^6.1.3",
     "i18next-chained-backend": "^3.0.2",
     "i18next-http-backend": "^1.3.2",
-    "i18next-icu": "^2.0.3",
+    "i18next-icu": "2.0.3",
```

Comment:
so.. fun story: we have dependency conflicts when i18next-icu: ^2.0.3 resolves to 2.1.0 because of their changed requirement of `intl-messageformat` from `9.4.6` to `10.2.4`.

Reply:
I wish package.json supported comments so these things could be called out more easily.